### PR TITLE
Cypress POC Bump

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Use Cypress instead of Testcafe
- Remove local cluster console tests override, as it wouldn't work anyway (and isn't needed when we use Cypress)

**Related issue(s)**
[Console](https://github.com/kyma-project/console/pull/1937)

### CODE REVIEW

**Comment by Wawrzyn321** [comment]
pre-master-kyma-gke-integration, pre-master-kyma-gke-upgrade passed, pre-master-kyma-integration timed out with console-web running.

**Comment by Wawrzyn321** [comment]
even with higher cpu, `console-web` suffocated other tests.

**Comment by Wawrzyn321** [comment]
now with reorganized tests structure.

**Comment by Wawrzyn321** [comment]
all console-web passed. Excellent. Moving on with more tests.

**Comment by Wawrzyn321** [comment]
magnificent.

**Comment by Wawrzyn321** [comment]
even _compass-integration_ has passed. Let's go again.

**Comment by Wawrzyn321** [comment]
it turns out bumping the memory was the culprit of timing out tests... I assumed they are failing because of timeout, but it they were failing because of cluster addons test (at least after removing memory bump changes). I'll try upping the Cypress timeout limit from 10s to 30s.

**Comment by Wawrzyn321** [comment]
Even though, tests on kyma integration are failing, sometimes randomly. I think we'd better of disabling them anyway.

**Comment by Wawrzyn321** [comment]
ohboyohboy

File: resources/console/charts/web/values.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ replicaCount: 1
 resources: {}
 test:
   acceptance:
-    enabled: false
+    enabled: true
```
Comment (parostatkiem) [inline resources/console/charts/web/values.yaml:39]: We could gain more stability with setting this to `false`...

**Review by parostatkiem — APPROVED** [review]

File: resources/console/charts/web/values.yaml (hunk 1 above)
Comment (Wawrzyn321) [reply resources/console/charts/web/values.yaml:39]: It would be the same stability you gain by adding child's training wheels to a motorcycle...

**Review by magicmatatjahu — APPROVED** [review]

**Comment by Wawrzyn321** [comment]
> THROUGH THE DARKNESS OF TESTCAFE'S PAST
> THE HASSELHOFF LONGS TO SEE
> ONE CHANTS OUT BETWEEN TWO COMMITS
> CYPRESS WALK WITH ME.

**Pull request merged**